src/components/filter/EventsFilterPanel.tsx

```diff
@@ -128,6 +142,13 @@ function EventsFilterPanel() {
 
 			let toggler: FilterRowTogglerConfig | null = null;
 
+			const autocompleteList = getArrayOfUniques(
+				autocompletes
+					.filter(item => item.filters[(filter.name as unknown) as keyof EventFilterState])
```

Comment:
`filter.name` is an object, how can you convert it to key?

did you mean to check if `filterName in item.filters` ?